src/App/DocumentObserver.h

```diff
@@ -244,6 +251,37 @@ class AppExport SubObjectT : public DocumentObjectT
 
     std::string getSubObjectPython(bool force=true) const;
 
+    /// Options used by normalize()
+    enum class NormalizeOption : uint8_t
+    {
+        /// Do not include sub-element reference in the output path
+        NoElement = 0x01,

```

Comment:
It may be just me, but I always get confused where the docstrings belong to somehow, so I would prefer blank lines separating the enum/docstring pairs.

Reply:
Yep, that's it. Thanks!